### PR TITLE
Fix some translations

### PR DESCRIPTION
**Pull request by xtreemtm**
God there was a lot of typos.
Changed Google+ references to Google
Translated cpp_blurb
Improved some lines

### CODE REVIEW

File: app/locale/uk.coffee [hunk 1]
```diff
@@ -1350,7 +1350,7 @@ module.exports = nativeDescription: "Українська", englishDescription:
     required_parameters: "Обов'язкові параметри"
     optional_parameters: "Необов'якові параметри"
     returns: "Повертає"
-    granted_by: "Забезпечує"
+    granted_by: "Забезпечує" # "Забезпечено" would be better in this context: "Granted by Some Item"
```
Comment (mrfinch) [inline app/locale/uk.coffee:1353]: if you think it's more appropriate, go ahead with the change rather than adding comment!!

**Review by mrfinch — COMMENTED** [review]
Thanks for making the changes!! Minor comment and we should be good to go

**Comment by ZJUSTKIDDING** [comment]
What do you mean by minor comment?
The commit message?

File: app/locale/uk.coffee (hunk 1 above)
Comment (ZJUSTKIDDING) [reply app/locale/uk.coffee:1353]: I left this for me to fix later, after verifying tge context. Going to look to this on weekends.

**Comment by xtreemtm** [comment]
Ok @mrfinch, fixed the string on line 1353. I think it's ready to merge.

**Pull request merged**